components/automate-ui/src/app/entities/userperms/userperms.requests.ts

```diff
@@ -18,19 +18,19 @@ export class UserPermsRequests {
   // Returns data for all *non-parameterized* endpoints
   // (i.e. endpoints like "/iam/v2/teams" but not "/iam/v2/users/{email}").
   public fetchAll(): Observable<UserPermsResponse> {
-    const url = `${env.auth_url}/introspect`;
+    const url = `${env.auth_v2_url}/introspect`;
```

Comment:
Finally, retarget the UI to use the new v2 endpoints.

Reply:
should this just be renamed `auth_url` (given our efforts to drop the "v2" wherever possible)?

Reply:
or `iam`? since all our APIs now live under `iam`

Reply:
Definitely--thanks!!